runtime/legion/legion_views.h

```diff
@@ -804,25 +836,29 @@ namespace Legion {
                                  const FieldMask &mask, 
                                  bool reading, bool restrict_out,
                                  const AddressSpaceID source,
-                                 std::set<RtEvent> &applied_events);
+                                 std::set<RtEvent> &applied_events,
+                                 PhysicalTraceInfo &trace_info);
       virtual ApEvent find_user_precondition(const RegionUsage &user,
                                            ApEvent term_event,
                                            const FieldMask &user_mask,
                                            Operation *op, const unsigned index,
                                            VersionTracker *version_tracker,
-                                           std::set<RtEvent> &applied_events);
+                                           std::set<RtEvent> &applied_events,
```

Comment:
Why is the applied_events now needed on some of these methods?

Reply:
What do you mean? I didn't add applied_events to this function.